src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -1490,30 +1532,35 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         leaveEventsOnly = isEventsOnlySpecified
                     End If
 
-                    If lookupResult.IsGood AndAlso currentResult.IsGood Then
+                    If lookupResult IsNot Nothing AndAlso lookupResult.IsGood AndAlso currentResult.IsGood Then
                         ' We have _another_ viable result while lookupResult is already viable. Use special interface merging rules.
                         MergeInterfaceLookupResults(lookupResult, currentResult, basesBeingResolved, leaveEventsOnly, useSiteDiagnostics)
                     Else
                         If currentResult.IsGood AndAlso leaveEventsOnly.HasValue Then
                             FilterSymbolsInLookupResult(currentResult, SymbolKind.Event, leaveInsteadOfRemoving:=leaveEventsOnly.Value)
                         End If
-                        lookupResult.MergePrioritized(currentResult)
+
+                        If lookupResult Is Nothing AndAlso currentResult.Kind <> LookupResultKind.Empty Then
```

Comment:
>currentResult.Kind <> LookupResultKind.Empty  [](start = 59, length = 45)

```!IsClear```?

Reply:
An empty RHS is a NOP in `MergePrioritized`. I can make the change if you wish for future-proofing or some other reason, but it would not change the current behavior.